src/answers.sql

```diff
@@ -1,15 +1,250 @@
 -- Your answers here:
 -- 1
+--
+SELECT
+	TYPE ACCOUNT_TYPE,
+	ROUND(SUM(MOUNT)::NUMERIC, 2) MONEY_SUM
+FROM
+	ACCOUNTS
+GROUP BY
+	TYPE;
 
+--
+--
+--
+--
 -- 2
+--
+SELECT
+	COUNT(*) AS COUNT_USER_AT_LEAST_2_CURRENT_ACCOUNT
+FROM
+	(
+		SELECT
+			U.ID
+		FROM
+			USERS U
+			INNER JOIN ACCOUNTS A ON U.ID = A.USER_ID
+		WHERE
+			A.TYPE = 'CURRENT_ACCOUNT'
+		GROUP BY
+			U.ID
+		HAVING
+			COUNT(U.ID) > 1
+	);
 
+--
+--
+--
+--
+--
 -- 3
+--
+SELECT
+	ID,
+	ACCOUNT_ID,
+	TYPE,
+	MOUNT
+FROM
+	ACCOUNTS
+ORDER BY
+	MOUNT DESC
+LIMIT
+	5;
 
+--
+--
+--
+--
 -- 4
+-- I MAKE SURE I USE EVERY ACCOUNT BY UNION ALL!
```

Comment:
I like this consideration. It's not implicit in the exercise, but it's good to have it!